src/index.tsx

```diff
@@ -14,7 +14,10 @@ import * as serviceWorker from "./serviceWorker";
 import { Theme } from "./ui/Theme/Theme";
 
 const authToken = getOauthTokenOrRedirect();
-const apiUrl = process.env.REACT_APP_API_URL!;
+
+const apiPath = process.env.REACT_APP_API_PATH!;
```

Comment:
Pas sûr que ce soit nécessaire de variabiliser le path de l'API. Ce n'est pas comme si on avait plusieurs versions de l'API pour l'instant.
```suggestion
const apiPath = "/api/v1";
```